docs/README-cmake.md

```diff
@@ -36,7 +36,7 @@ This will build the static and dynamic versions of SDL in the `~/build` director
 Installation can be done using:
 
 ```sh
-cmake --install .        # '--install' requires CMake 3.15, or newer
+cmake --install . --prefix=`pwd`        # '--install' requires CMake 3.15, or newer

```

Comment:
The build directory is, imho, a bad suggestion for people to install SDL to.
Doing that will mix build folders (`CMakeFiles`) and `include`/`lib`/`share`/...

How about this?
```suggestion
# '--install' requires CMake 3.15, or newer
cmake --install . --prefix="$HOME/.local"        # Install SDL to `$HOME/.local`
```

Reply:
Either that or  `cmake --install .local --prefix="$HOME"` to drive the ...point...home... :)

Reply:
It needs to be `--install .`.  We're assuming the current working directory is the build folder.
```
       --install <dir>
              Project binary directory to install. This is required and must be first.

```

Reply:
Right- silly me. Thank you for the review.

Reply:
This line has not been changed yet.